draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -167,12 +167,12 @@
 
 		<date year="2021"/>
 		<abstract>
-		      <t>   Link State PDU flooding rates in use are much slower than what modern
+		      <t>   Current Link State Protocol Data Unit (PDU) flooding rates are much slower than what modern
    networks can support.  The use of IS-IS at larger scale requires
    faster flooding rates to achieve desired convergence goals.  This
-   document discusses the need for faster flooding, the issues of concern
-   when using faster flooding, and two candidate algorithms to achieve
-   faster flooding. It also defines protocol extensions to signal parameters of the LSPs receiver.
+   document discusses the need for faster flooding, the issues around
+   faster flooding, and some example approaches to achieve
+   faster flooding. It also defines protocol extensions to signal parameters of the LSP receiver.
```

Comment:
"It also defines protocol extensions that allow routers to signal parameters which may be of use in achieving faster flooding."

Note: I do not want to limit things which can be advertised to receiver parameters - maybe someone will come up with a Tx parameter that is useful to a receiver.

Reply:
Text is maleable.  It does not limit.  If we choose to add a transmit parameter, we can also modify this text.